Acme/Certificate/Extractor/CertificateExtractor.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the ACME PHP library.
+ *
+ * (c) Titouan Galopin <galopintitouan@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace AcmePhp\Bundle\Acme\Certificate\Extractor;
+
+use AcmePhp\Bundle\Exception\ParsingCertificateException;
+use AcmePhp\Core\Ssl\Certificate;
+
+/**
+ * Extract data from certificate file.
+ *
+ * @author Jérémy Derussé <jeremy@derusse.com>
+ */
+class CertificateExtractor implements ExtractorInterface
```

Comment:
Imo, it read nothing. Maybe parser is a proper word.

But I was thinking to extractors like in https://github.com/symfony/symfony/tree/master/src/Symfony/Component/PropertyInfo/Extractor